docs/design/using-office-ui-fabric-react.md

```diff
@@ -63,13 +63,20 @@ After you complete the wizard, the generator creates the project and installs su
         npm start
         ```
 
-    - To test your add-in in Word on a browser, run the following command in the root directory of your project. When you run this command, the local web server will start (if it's not already running).
+    - To test your add-in in Word on a browser, run the following command in the root directory of your project. When you run this command, the local web server will start (if it's not already running). Replace "{url}" with the URL of a Word document on your OneDrive or a SharePoint library to which you have permissions.
```

Comment:
Suggestion to remove the parenthetical statement.
```suggestion
    - To test your add-in in Word on a browser, run the following command in the root directory of your project. When you run this command, the local web server starts, if it's not already running. Replace "{url}" with the URL of a Word document on your OneDrive or a SharePoint library to which you have permissions.
```

Reply:
Please note that this comes up several times. If you choose to take this suggestion, please run a find/replace.